modules/apps/site/site-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Site API
 Bundle-SymbolicName: com.liferay.site.api
-Bundle-Version: 23.0.1
+Bundle-Version: 23.1.0
```

Comment:
@SamZiemer, wouldn't it be better to name the commit baseline instead of buildService?

Reply:
fixed in latest push